src/core/Radio/index.js

```diff
@@ -11,19 +10,15 @@ class Radio extends React.Component {
     }
 
     render() {
-        let icoColor = this.props.disabled ? 'grey-light' : 'grey'
-        let icoName
-
-        if (this.props.checked) {
-            icoName = 'radio_button_checked'
-        } else {
-            icoName = 'radio_button_unchecked'
-        }
-
+        let color = this.props.disabled ? 'grey-light' : 'grey'
         const icon = (
             <Icon
-                name={icoName}
-                className={`${icoColor} ${this.props.status}`}
+                name={
+                    this.props.checked
+                        ? 'radio_button_checked'
```

Comment:
I see we are missing a state here, a checked radio button should be `secondary-light`.